external/fv3fit/fv3fit/pytorch/graph_config.py

```diff
@@ -0,0 +1,64 @@
+import dataclasses
+from typing import Callable
+import torch.nn.functional as F
+import torch
+import torch.nn as nn
+from dgl.nn.pytorch import SAGEConv
+
+
+@dataclasses.dataclass
+class GraphNetworkConfig:
+    """
+    Attributes:
+        in_feats: number of input features
+        out_feats: number of output features
+        n_hidden: maximum number of hidden channels
+        num_step: number of U-net blocks
```

Comment:
These are not u-net blocks currently, and they aren't "steps" either - could you update the description, and change this name to something more descriptive of its use like "num_blocks"?